daprdocs/content/en/developing-applications/building-blocks/workflow/howto-manage-workflow.md

```diff
@@ -23,14 +23,26 @@ OrderPayload input = new OrderPayload("Paperclips", 99.95);
 Dictionary<string, string> workflowOptions; // This is an optional parameter
 CancellationToken cts = CancellationToken.None;
 
-// Start the workflow. This returns back a "WorkflowReference" which contains the instanceID for the particular workflow instance.
-WorkflowReference startResponse = await daprClient.StartWorkflowAsync(orderId, workflowComponent, workflowName, input, workflowOptions, cts);
+// Start the workflow. This returns back a "StartWorkflowResponse" which contains the instanceID for the particular workflow instance.
+StartWorkflowResponse startResponse = await daprClient.StartWorkflowAsync(orderId, workflowComponent, workflowName, input, workflowOptions, cts);
 
 // Get information on the workflow. This response will contain information such as the status of the workflow, when it started, and more!
 GetWorkflowResponse getResponse = await daprClient.GetWorkflowAsync(orderId, workflowComponent, workflowName);
 
 // Terminate the workflow
-await daprClient.TerminateWorkflowAsync(instanceId, workflowComponent);
+await daprClient.TerminateWorkflowAsync(orderId, workflowComponent, cts);
+
+// Raise an event (an incoming purchase order) that your workflow will wait for. This returns the item waiting to be purchased.
+await daprClient.RaiseWorkflowEventAsync(orderId, workflowComponent, workflowName, input, cts);
+
+// Pause
+await daprClient.PauseWorkflowAsync(orderId, workflowComponent, cts);
```

Comment:
I saw a discussion and code that said this was going to be called suspend, instead of pause. This code https://github.com/dapr/dapr/pull/6106/files. Is that correct?

Reply:
@RyanLettieri @cgillum can you confirm?

Reply:
It was my understanding that the code was going to reflect Pause/Resume rather than Suspend/Resume. However, the value returned to the user for the status when they call GET on a workflow instance that has been paused is 'Suspended'. I think we should leave it as is for now and potentially revisit later on.

Reply:
Unfortunately, there's an inconsistency which I think we missed in the implementation. The APIs are all called "Pause" but the workflow status is changed to "Suspended", as Ryan mentioned.

Reply:
We could change the `GetWorkflow` API handler to return "PAUSED" instead of "SUSPENDED". I think we'd need to take this up with the Dapr maintainers as I expect there will be some angst about making changes this late in the milestone.